pangocairo/src/font_map.rs

```diff
@@ -1,38 +0,0 @@
-// Take a look at the license at the top of the repository in the LICENSE file.
-
-use crate::FontMap;
-use glib::object::IsA;
-use glib::translate::*;
-
-pub trait FontMapExtManual {
-    #[doc(alias = "pango_cairo_font_map_get_font_type")]
-    #[doc(alias = "get_font_type")]
-    fn font_type(&self) -> cairo::FontType;
-}
-
-impl<O: IsA<FontMap>> FontMapExtManual for O {
-    fn font_type(&self) -> cairo::FontType {
-        unsafe { ffi::pango_cairo_font_map_get_font_type(self.as_ref().to_glib_none().0).into() }
-    }
-}
-
-impl FontMap {
-    #[doc(alias = "pango_cairo_font_map_new_for_font_type")]
-    #[doc(alias = "new_for_font_type")]
-    pub fn for_font_type(fonttype: cairo::FontType) -> Option<pango::FontMap> {
-        unsafe { from_glib_full(ffi::pango_cairo_font_map_new_for_font_type(fonttype.into())) }
-    }
-
-    #[allow(clippy::new_ret_no_self)]
-    #[doc(alias = "pango_cairo_font_map_new")]
-    pub fn new() -> Option<pango::FontMap> {
-        unsafe { from_glib_full(ffi::pango_cairo_font_map_new()) }
-    }
-
-    #[doc(alias = "pango_cairo_font_map_set_default")]
-    pub fn set_default(font_map: Option<Self>) {
```

Comment:
`set_default()` should probably stay an associated function. `font_map.set_default()` seems strange compared to `pangocairo::FontMap::set_default(&font_map)`. But document that in Gir.toml then

Reply:
But it should take a `font_map: Option<&Self>` instead of `Option<Self>`.

Also it should be an associated function because of the `Option` part.